addons/website_account/models/res_config_settings.py

```diff
@@ -0,0 +1,20 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+from odoo import api, models, fields
+
+from werkzeug.urls import url_join
+
+class ResConfigSettings(models.TransientModel):
+    _inherit = 'res.config.settings'
+
+    terms_url = fields.Char(compute='_compute_terms_url', string="URL", help="A preview will be available at this URL.")
+
+    @api.depends('website_id')
+    def _compute_terms_url(self):
+        for record in self:
+            record.terms_url = url_join(record.website_id.get_base_url(), '/terms')
+
+    def action_update_terms(self):
+        self.ensure_one()
+        return self.env["website"].get_client_action('/terms', True)
```

Comment:
I don't like the code colliding on that model when both module gets installed but I don't really know what to do about it.
It's probably not a problem as both methods are doing exactly the same thing.

@qdp-odoo Do you think we should change the methods name? Or we don't care (Or should we break the stable policy and for the customer to update?)

Reply:
yes I know it looks ugly but it's the only way not to break stable I guess